moj-node/server/routes/search.js

```diff
@@ -10,17 +9,17 @@ module.exports = function createSearchRouter({ searchService, logger }) {
     postscript: false,
   };
 
-  router.use(bodyParser());
-
   router.get('/', async (req, res, next) => {
     logger.info('GET /search');
 
+    const { establishmentId } = req.app.locals.envVars;
+
     try {
       let results = [];
-      const query = req.query.search;
+      const { query } = req.query;
 
       if (query) {
```

Comment:
This logic could live in the service later
`  if (query) {`
Cleaning up the route from logic

Reply:
Moved into service